src/lib/layouts/Sidebar.svelte

```diff
@@ -33,7 +33,7 @@
 	}
 </script>
 
-<nav class="aw-side-nav">
+<nav class="aw-side-nav" aria-label="Side Navigation">
```

Comment:
We can omit the term `Navigation` here. A screen reader will read that it is a navigation element, and the aria label, so that would just be annoying to the user 😄